File: MaybeNewSolutions/题目1066：字符串排序.java

import java.util.Arrays;
import java.util.Scanner;

/** 
 * @ClassName: Main1046 
 * @Description: TODO 
 * @author wangzq
 * @date 2013-8-7 下午04:00:39 
 * 
 * @version 3.0.0 
 */
public class Main{
	/*
	 * 1066
	 */
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		while (scanner.hasNext()) {
			String a = scanner.nextLine();
			char [] arrayA = a.toCharArray();
			Arrays.sort(arrayA);
			String result = "";
			for (int i = 0; i < arrayA.length; i++) {
				result += arrayA[i];
				
			}
			System.out.println(result);
		}
	}
	
}
/**************************************************************
	Problem: 1066
	User: wzqwsrf
	Language: Java
	Result: Accepted
	Time:1840 ms
	Memory:174024 kb
****************************************************************/


                        

File: C++/题目1066：字符串排序.cpp

// 题目1066：字符串排序

/**
 * @author:wangzq
 * @email:wangzhenqing1008@163.com
 * @date:2015-06-30 11:01:52
 * @url:http://ac.jobdu.com/problem.php?pid=1066
 */

import java.util.Arrays;
#include <stdio.h>
#include <stdlib.h>
#include <algorithm>
#include <string>
#include <string.h>

using namespace std;

int main() {
    char s[202];
    while (scanf("%s", s) != EOF) {
        int len = strlen(s);
        sort(s, s + len);
        printf("%s\n", s);
    }
    return 0;
}
/**************************************************************
Problem: 1066
User: wangzhenqing
Language: C++
Result: Accepted
Time:90 ms
Memory:1020 kb
****************************************************************/


                        

